src/functions-and-arrays.js

```diff
@@ -52,14 +103,31 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  if(arr.length === 0){
+    return null
+  }else{
+    let res = []
+    arr.forEach(rep => {
+      if ( res.indexOf(rep) == -1)
```

Comment:
Attention to the use of double equal signs. It is better pratice if you use '==='.